custom/conf/app.example.ini

```diff
@@ -171,6 +171,11 @@ RUN_MODE = ; prod
 ;; Will default to the PER_WRITE_PER_KB_TIMEOUT.
 ;SSH_PER_WRITE_PER_KB_TIMEOUT = 30s
 ;;
+;; Redirect path to username / organisation
+;; Only applicable when repository path is missing either user or organization.
+;; Useful when taking over older repository servers "gitserver:repo" -> "gitserver:redirectorg/repo"
+;SSH_REDIRECT_PATH = team22
```

Comment:
this should be with default value (empty not team22)

Reply:
Thanks, will change accordingly.